packages/node-core/src/configure/SchemaMigration.service.ts

```diff
@@ -343,6 +261,7 @@ export class SchemaMigrationService {
     dbSchema: string,
     blockHeight: number,
     _flushCache: (flushAll?: boolean) => Promise<void>,
+    updateCacheModels: (model: ModelStatic<Model<any, any>>[]) => void,
```

Comment:
`updateCacheModels` -> `updatedModels`. This can be generic, the migration service shouldn't be concerned with how these new models are going to be used.